src/main/resources/default/assets/scripts/blob-upload-url-popover.js

```diff
@@ -12,16 +12,19 @@
 function initUrlUploadPopover(_triggerElement, _uploadContainer, _resetButton, _fileNameContainer, i18n, updateCallback) {
     const body = buildBody(_triggerElement, _uploadContainer, _resetButton);
     const container = buildContainer(_triggerElement.parentElement);
-    $(_triggerElement).popover({
-        html: true,
-        trigger: 'manual',
-        content: body._container,
-        container: container,
-        placement: 'top'
-    });
 
-    _triggerElement.addEventListener('click', function() {
-        $(_triggerElement).popover('toggle');
+    if (!_triggerElement.popover) {
+        _triggerElement.popover = new bootstrap.Tooltip(_triggerElement, {
+            html: true,
+            trigger: 'manual',
+            content: body._container,
+            container: container,
+            placement: 'top'
+        });
+    }
+
+    _triggerElement.addEventListener('click', () => {
```

Comment:
Don't know if we should keep ie11 compatible syntax here as the file may be shared with wondergem code? E.g. S2 may use this in the FE

Reply:
Thank @fhaScireum … There is a second folder, `src/main/resources/default/assets/javascript` (i.e. `javascript` instead of `scripts`) that contains code for _Wondergem_ … I also checked the other scripts in the `scripts` directory and they are only used from within _Tycho._

Reply:
thx @ both